### PR TITLE
pyqt5: enable qtwebkit for clang

### PR DESCRIPTION
**Pull request by raedrizqie**

### CODE REVIEW

File: mingw-w64-pyqt5/PKGBUILD [hunk 1]
```diff
@@ -9,20 +9,20 @@ conflicts=("${MINGW_PACKAGE_PREFIX}-python3-${_realname}"
...
+depends=("${MINGW_PACKAGE_PREFIX}-python"
          "${MINGW_PACKAGE_PREFIX}-python-pyopengl"
          "${MINGW_PACKAGE_PREFIX}-python-ipykernel"
          "${MINGW_PACKAGE_PREFIX}-python-pygments"
          "${MINGW_PACKAGE_PREFIX}-python-qtpy"
-         "${MINGW_PACKAGE_PREFIX}-python"
          "${MINGW_PACKAGE_PREFIX}-pyqt5-sip"
-         "${MINGW_PACKAGE_PREFIX}-qt5-base")
+         "${MINGW_PACKAGE_PREFIX}-qt5-base"
+         "${MINGW_PACKAGE_PREFIX}-qtwebkit")
```
Comment (MehdiChinoune) [inline mingw-w64-pyqt5/PKGBUILD:25]: PLease, remove it from dependencies.

Comment (raedrizqie) [reply mingw-w64-pyqt5/PKGBUILD]: done

**Comment by raedrizqie** [comment]
does pyqt5 really working without any patching?

**Comment by MehdiChinoune** [comment]
> does pyqt5 really working without any patching?

Yes, It is working. I have used it many times.
We didn't see bug reports about it.

**Comment by raedrizqie** [comment]
ok.. because spyder failed after the loading screen.. will investigate further

**Comment by MehdiChinoune** [comment]
> ok.. because spyder failed after the loading screen.. will investigate further

I have some failures with recent versions (>5.15.4) with some io operations. Does spyder read process some project files or something like that.
Try to build it against PyQt 5.15.4 and see if it still occurs. If it solves the issue I will push a fix for it.

**Comment by raedrizqie** [comment]
i got this error:

>  File "C:/msys64/clang64/lib/python3.9/site-packages/spyder/widgets/browser.py", line 367, in setHtml
    self.focusProxy().installEventFilter(self)
AttributeError: 'NoneType' object has no attribute 'installEventFilter'
Traceback (most recent call last):
  File "C:/msys64/clang64/lib/python3.9/site-packages/spyder/app/utils.py", line 281, in create_window
    main.setup()
  File "C:/msys64/clang64/lib/python3.9/site-packages/spyder/app/mainwindow.py", line 875, in setup
    PLUGIN_REGISTRY.register_plugin(self, PluginClass,
  File "C:/msys64/clang64/lib/python3.9/site-packages/spyder/api/plugin_registration/registry.py", line 274, in register_plugin
    instance = self._instantiate_spyder5_plugin(
  File "C:/msys64/clang64/lib/python3.9/site-packages/spyder/api/plugin_registration/registry.py", line 152, in _instantiate_spyder5_plugin
    plugin_instance = PluginClass(main_window, configuration=CONF)
  File "C:/msys64/clang64/lib/python3.9/site-packages/spyder/api/plugins/new_api.py", line 890, in __init__
    super().__init__(parent, configuration=configuration)
  File "C:/msys64/clang64/lib/python3.9/site-packages/spyder/api/plugins/new_api.py", line 301, in __init__
    container.setup()
  File "C:/msys64/clang64/lib/python3.9/site-packages/spyder/plugins/help/widgets.py", line 478, in setup
    self.show_intro_message()
  File "C:/msys64/clang64/lib/python3.9/site-packages/spyder/plugins/help/widgets.py", line 801, in show_intro_message
    self.set_rich_text_html(usage(title, intro_message,
  File "C:/msys64/clang64/lib/python3.9/site-packages/spyder/plugins/help/widgets.py", line 746, in set_rich_text_html
    self.rich_text.set_html(html_text, base_url)
  File "C:/msys64/clang64/lib/python3.9/site-packages/spyder/plugins/help/widgets.py", line 191, in set_html
    self.webview.setHtml(html_text, base_url)
  File "C:/msys64/clang64/lib/python3.9/site-packages/spyder/widgets/browser.py", line 367, in setHtml
    self.focusProxy().installEventFilter(self)
AttributeError: 'NoneType' object has no attribute 'installEventFilter'
>
>During handling of the above exception, another exception occurred:
>
>Traceback (most recent call last):
  File "C:\msys64\clang64\bin\spyder-script.pyw", line 33, in <module>
    sys.exit(load_entry_point('spyder==5.1.5', 'gui_scripts', 'spyder')())
  File "C:/msys64/clang64/lib/python3.9/site-packages/spyder/app/start.py", line 236, in main
    mainwindow.main(options, args)
  File "C:/msys64/clang64/lib/python3.9/site-packages/spyder/app/mainwindow.py", line 1992, in main
    mainwindow = create_window(MainWindow, app, splash, options, args)
  File "C:/msys64/clang64/lib/python3.9/site-packages/spyder/app/utils.py", line 283, in create_window
    if main.console is not None:
  File "C:/msys64/clang64/lib/python3.9/site-packages/spyder/app/mainwindow.py", line 1088, in __getattr__
    return self.get_plugin(self._INTERNAL_PLUGINS_MAPPING[attr])
  File "C:/msys64/clang64/lib/python3.9/site-packages/spyder/app/mainwindow.py", line 165, in get_plugin
    raise SpyderAPIError(f'Plugin "{plugin_name}" not found!')
spyder.api.exceptions.SpyderAPIError: Plugin "internal_console" not found!

maybe spyder cannot find some plugins?

**Comment by MehdiChinoune** [comment]
I don't see anything related to pyqt5 in the error log.

**Comment by raedrizqie** [comment]
> I don't see anything related to pyqt5 in the error log.

ok.. its spyder that needs to be patched then

**Pull request merged**